.github/workflows/android.yaml

```diff
@@ -141,16 +137,16 @@ jobs:
     - name: Instrumentation Tests
       uses: reactivecircus/android-emulator-runner@f0d1ed2dcad93c7479e8b2f2226c83af54494915 # pin@v2.32
       with:
-        channel: beta
+        channel: stable
         force-avd-creation: false
         api-level: 21
         target: default
         ram-size: 2048M
         arch: x86_64
-        disk-size: 4096M
+        disk-size: 6144M
         profile: Nexus 6
         disable-animations: true
-        emulator-options: -no-snapshot-save -no-window -accel on -gpu swiftshader_indirect -noaudio -no-boot-anim -camera-back none
```

Comment:
is this related to the kvm change above? also wonder if this would make it slower? (not that these take that long anyway_)